src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDatabaseMetaData.java

```diff
@@ -638,10 +638,10 @@ private static String EscapeIDName(String inID) throws SQLServerException {
             PreparedStatement pstmt = (SQLServerPreparedStatement) this.connection.prepareStatement(spColumnsSql);
             pstmt.closeOnCompletion();
             try {
-                pstmt.setString(1, (null != table && !table.isEmpty()) ? table : "%");
-                pstmt.setString(2, (null != schema && !schema.isEmpty()) ? schema : "%");
+                pstmt.setString(1, (null != table && !table.isEmpty()) ? EscapeIDName(table) : "%");
+                pstmt.setString(2, (null != schema && !schema.isEmpty()) ? EscapeIDName(schema) : "%");
                 pstmt.setString(3, (null != catalog && !catalog.isEmpty()) ? catalog : this.connection.getCatalog());
-                pstmt.setString(4, (null != col && !col.isEmpty()) ? col : "%");
+                pstmt.setString(4, (null != col && !col.isEmpty()) ? EscapeIDName(col) : "%");
```

Comment:
There is an `else` block below that handles this logic with Azure DW. That section needs similar changes.

Reply:
Thanks. Done